tests/ci/README.md

```diff
@@ -49,12 +49,6 @@ CodeBuild|gcc 7.3.1|x86-64|AL2
 CodeBuild|gcc 7.3.1|aarch64|AL2
 CodeBuild|gcc 7.4.4|x86-64|Ubuntu 18.04
 CodeBuild|gcc 8.3.0|x86-64|Ubuntu 19.04
-CodeBuild|gcc 9.2.1|x86-64|Ubuntu 19.10
```

Comment:
Should we put a note in the README why we don't support these?

Reply:
I think no. GCC 6 and other minor versions are not listed here.

Reply:
Fair. It just might look odd to someone seeing it the first time with 9.x missing.

We should really somehow link to this table in the root README.